client/src/components/Hamburger/index.js

```diff
@@ -45,38 +45,38 @@ const HamburgerInner = styled.span`
   margin-top: -2px;
 
   transition-duration: 0.13s;
-  transition-delay: ${p => (p.isOpen === true ? '0.22s;' : '0.13s')};
+  transition-delay: ${p => (p.isOpen ? '0.22s;' : '0.13s')};
```

Comment:
With styled components if you put the units at the end you do not have to make everything in your ternary a string
```suggestion
  transition-delay: ${p => (p.isOpen ? 0.22 : 0.13)}s;
```

Reply:
good tip